packages/icons/karma.conf.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
`/*` only, this is not documentation

Reply:
💯 for enabling karma tests!

Reply:
I copied it from `karma.conf.js` in `core`, which uses `/**` in the copyright header. :confused: